CONTRIBUTING.md

```diff
@@ -0,0 +1,40 @@
+Software Carpentry is an open source project,
+and we welcome contributions of all kinds:
+new lessons,
+fixes to existing material,
+bug reports,
+and reviews of proposed changes are all equally welcome.
+
+By contributing,
+you are agreeing that Software Carpentry may redistribute your work under
+[these licenses](LICENSE.html).
+
+## Getting Started
+
+1.  We use GitHub flow to manage changes,
+    which is explained in the chapter [Contributing to a Project](http://git-scm.com/book/en/v2/GitHub-Contributing-to-a-Project)
+    in Scott Chacon's book *Pro Git*.
+
+2.  For our lessons,
+    you should branch from and submit pull requests against the `gh-pages` branch.
+
+3.  When editing lesson pages, you need only commit changes to the Markdown source files.
+
+4.  If you're looking for things to work on,
+    please see the current list of issues
+    for [this repository](https://github.com/swcarpentry/lesson-template/issues),
```

Comment:
I would suggest to use reference-style links, e.g. `[this repository][github-address]`, to make easy for people find the line they should edit. If this suggestion was accepted will be nice to change `LAYOUT.md` to

~~~
8.  `CONTRIBUTING.md`: instructions for people who want to contribute.
    This file should be edited to replace references the link associate with `github-address`
    at the end of the file to reference your lesson's repository.
~~~